# This message holds the position information of a single detected traffic light.

float64[] singlepos # size ( 9 items )